sauce/features/general/edit-account-button/settings.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  name: 'EditAccountButton',
+  type: 'select',
+  default: '0',
+  section: 'general',
+  title: 'Edit Account Button Position',
+  description: 'Allows you to move or hide the edit account button to help prevent accidentally clicking on it.',
+  options: [
+    { name: 'Default', value: '0' },
+    { name: 'Left of name', value: '1' },
+    { name: 'Hidden', value: '2' }
```

Comment:
I seem to recall the earlier version had some text about right-clicking to edit the account. Is that still valid with the refactor? If so, I'd add text to that effect.

Reply:
you can right-click to edit regardless of this feature being on but I can still add it